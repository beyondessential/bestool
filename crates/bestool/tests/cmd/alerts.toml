bin.name = "bestool"
args = "tamanu --root ./tamanu alerts --interval 1w --dir ./alerts --dry-run"
