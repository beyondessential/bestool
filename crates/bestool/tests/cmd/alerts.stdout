-------------------------------
Alert: ./alerts/sql.yml
Recipients: test@example.com
Subject: Tamanu Alert - 1970-01-01 00:00:00 UTC
Body: <p>Automated alert! There have been 5 jobs
with errors in the past 1w. Here are the first 2:</p>
<ul>
<li>
<p>foo: err</p>
</li>
<li>
<p>bar: err</p>
</li>
</ul>

